src/ui/map.js

```diff
@@ -1833,12 +1846,7 @@ class Map extends Camera {
      * @memberof Map
      */
     get repaint(): boolean { return !!this._repaint; }
-    set repaint(value: boolean) {
```

Comment:
I think this repaint change and also the `isMoving` change above are also unintended?